### ISSUE TITLE
Impossible to use the GetFeatureInfo as long as the plugin is loaded

### ISSUE THREAD

**Issue by MaelREBOUX** [sigrennesmetropole/geor_urbanisme_mapstore#62]
It is currently impossible to get the information panel on a feature of any layer in the map as long as the Urbanimse plugin is loaded.

See below :

![ms-urbanisme-gfi-layer](https://user-images.githubusercontent.com/6370443/206453614-47fce1e5-778e-4c9f-8981-32a9aa05d4aa.gif)

Expected behavior
If the "NRU" tool is desactivated (button not green in the toolbar) : we should be able ton clic on an object and get its information form.

@catmorales  could you please confirm that behavior ?

**Comment by catmorales** [comment]
@MaelREBOUX Yes I do, it's the current behaviour. We have to evaluate this enhancement.
